src/DssSpell.t.sol

```diff
@@ -614,369 +604,426 @@ contract DssSpellTest is DssSpellTestBase {
+        assertEq(rwa015AOutputConduit.can(pauseProxy),        0, "OutputConduit/pause-proxy-hoped");
+        assertEq(rwa015AOutputConduit.can(RWA015_A_OPERATOR), 1, "OutputConduit/operator-not-hope");
+        assertEq(rwa015AOutputConduit.may(pauseProxy),        0, "OutputConduit/pause-proxy-not-mated");
+        assertEq(rwa015AOutputConduit.may(RWA015_A_OPERATOR), 1, "OutputConduit/operator-not-mate");
+        assertEq(rwa015AOutputConduit.bud(RWA015_A_CUSTODY),  1, "OutputConduit/destination-address-not-whitelisted-for-pick");
+
+        assertEq(rwa015AOutputConduit.quitTo(), address(rwa015AUrn), "OutputConduit/quit-to-not-urn");
+
+        assertEq(rwa015AInputConduitUrn.wards(pauseProxy),      1, "InputConduitUrn/ward-pause-proxy-not-set");
+        assertEq(rwa015AInputConduitUrn.wards(address(esm)),    1, "InputConduitUrn/ward-esm-not-set");
+        assertEq(rwa015AInputConduitUrn.may(pauseProxy),        0, "InputConduitUrn/pause-proxy-mated");
+        assertEq(rwa015AInputConduitUrn.may(RWA015_A_OPERATOR), 1, "InputConduitUrn/operator-not-mate");
+
+        assertEq(rwa015AInputConduitUrn.quitTo(), RWA015_A_CUSTODY, "InputConduitUrn/quit-to-not-set");
+
+        assertEq(rwa015AInputConduitJar.wards(pauseProxy),      1, "InputConduitJar/ward-pause-proxy-not-set");
+        assertEq(rwa015AInputConduitJar.wards(address(esm)),    1, "InputConduitJar/ward-esm-not-set");
+        assertEq(rwa015AInputConduitJar.may(pauseProxy),        0, "InputConduitJar/pause-proxy-mated");
+        assertEq(rwa015AInputConduitJar.may(RWA015_A_OPERATOR), 1, "InputConduitJar/operator-not-mate");
+
+        assertEq(rwa015AInputConduitJar.quitTo(), RWA015_A_CUSTODY, "InputConduitJar/quit-to-not-set");
+
+        assertEq(rwa015AJoin.wards(address(esm)),        1, "Join/ward-esm-not-set");
+        assertEq(rwa015AJoin.wards(address(rwa015AUrn)), 1, "Join/ward-urn-not-set");
+
+        assertEq(rwa015AUrn.wards(pauseProxy),      1, "Urn/ward-pause-proxy-not-set");
+        assertEq(rwa015AUrn.wards(address(esm)),    1, "Urn/ward-esm-not-set");
+        assertEq(rwa015AUrn.can(pauseProxy),        0, "Urn/pause-proxy-hoped");
+        assertEq(rwa015AUrn.can(RWA015_A_OPERATOR), 1, "Urn/operator-not-hoped");
     }
 
-    function testRWA014_INTEGRATION_BUMP() public {
+    function testRWA015_INTEGRATION_BUMP() public {
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         GodMode.setWard(address(oracle), address(this), 1);
 
-        (, address pip, , ) = oracle.ilks("RWA014-A");
+        (, address pip, , ) = oracle.ilks("RWA015-A");
 
-        assertEq(DSValueAbstract(pip).read(), bytes32(500 * MILLION * WAD), "RWA014: Bad initial PIP value");
+        assertEq(DSValueAbstract(pip).read(), bytes32(2_500_000 * WAD), "RWA015-A: Bad initial PIP value");
 
-        oracle.bump("RWA014-A", 510 * MILLION * WAD);
+        oracle.bump("RWA015-A", 1_280 * MILLION * WAD);
```

Comment:
This is completely arbitrary, nothing will be "ruined" because we're bumping to a different value. The goal of this test is to assert whether we can update the price or not, the actual value doesn't matter.

Reply:
OK so if it's arbitrary then change it back to the archive pattern of 500M.

Spell team will not allow a change from archive pattern without a good reason, and by your explanation there is no good reason for it to be 1280 instead of 500 (or vice versa) so the existing precedent of +10M of the existing value is what we follow.